Source/Mosa.Demo.SVGAWorld.x86/Plugs/ConsolePlug.cs

```diff
@@ -4,7 +4,7 @@
 using Mosa.Kernel.x86;
 using Mosa.Runtime.Plug;
 
-namespace Mosa.Plug.Korlib.System.x86;
```

Comment:
korlib plugs should stay in `Mosa.Plug.Korlib.*`

Reply:
The issue with that is that they're less flexible. I think it's best to implement some plugs directly in the kernel instead of in Mosa.Plug.Korlib.*. For example, Environment.FailFast(), it's better to do so because the nyou can disable the display driver (if any), and you can't just guess that when it's in Mosa.Plug.Korlib.*.